sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py

```diff
@@ -0,0 +1,96 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_basics_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to use evaluation operations from
+    the Azure Evaluation service using a asynchronous client.
+
+USAGE:
+    python sample_evaluation_async.py
+
+    Before running the sample:
+
+    pip install azure.ai.projects azure-identity
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+import asyncio
+import time
+import os
+
+from azure.ai.projects.aio import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import Evaluation, Dataset, EvaluatorConfiguration, ConnectionType
+from azure.ai.evaluation import F1ScoreEvaluator, RelevanceEvaluator, ViolenceEvaluator
+
+
+async def main():
+    # Create an Azure AI Client from a connection string, copied from your AI Studio project.
+    # At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+    # Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
```

Comment:
I removed the above comments in my samples. And I think Agents samples as well. There will be is plenty of info on this in the package README.md.

Reply:
Removed it.